akka-http/src/main/scala/akka/http/javadsl/server/directives/HeaderDirectives.scala

```diff
@@ -20,13 +20,9 @@ import akka.stream.ActorMaterializer
 import scala.reflect.ClassTag
 import scala.util.{Failure, Success}
 
-/**
- * FIXME When implementing custom headers in Java, just extend akka.http.scaladsl.model.headers.CustomHeader,
- * since the java API's CustomHeader class does not implement render().
- */
 abstract class HeaderDirectives extends FutureDirectives {
 
-  type ScalaHeaderMagnet = HeaderMagnet[akka.http.scaladsl.model.HttpHeader]
+  private type ScalaHeaderMagnet = HeaderMagnet[akka.http.scaladsl.model.HttpHeader]
```

Comment:
Yeah, it's just for the insides of this class AFAIR, also it's a Java API so not sure what it would mean if public